src/dsp_tools/commands/validate_data/sparql/cardinality_shacl.py

```diff
@@ -88,7 +87,9 @@ def _construct_1_cardinality(onto_graph: Graph) -> Graph:
     
       ?propRestriction knora-api:isEditable true .
       FILTER NOT EXISTS { ?propRestriction knora-api:isLinkValueProperty true }
-    
+      FILTER (?class NOT IN (
```

Comment:
Because we include knora-api now, these cause cardinality violations because we don't construct the triples for them yet. Once we implement these we can remove this filter.